packages/@featherds/utils/setTimeout.ts

```diff
@@ -0,0 +1,17 @@
+const _setTimeout = function (callback: Function, timeout: number) {
+  if (!window) {
+    return setTimeout(callback, timeout);
+  } else {
+    return window.setTimeout(callback, timeout);
+  }
+};
+
+const _clearTimeout = function (id: number) {
+  if (!window) {
+    return clearTimeout(id);
+  } else {
+    return window.clearTimeout(id);
+  }
+};
+
+export { _setTimeout, _clearTimeout };
```

Comment:
My only real concern is the _ prefix which I always associated with private methods otherwise this all looks good

Reply:
Happy to change this to something more appropriate - this was chosen to avoid any ambiguity/name collisions.